content/1.7/back-office/shipping/carriers/Add-edit-carrier.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Add/edit carrier
+weight: 1
+---
+
+# General settings (1st step)
```

Comment:
Both call to action logo and steps are described at the begining. Could you please remove them from this section?